guide/plg_interfaces.tex

```diff
@@ -638,7 +638,7 @@ \section{Solar System Editor Plugin}
 The second tab lists all currently loaded minor bodies (see figure~\ref{fig:SolarSystemEditor:SolarSystem}).  It is recommended
 to remove old entries of yesteryear's comets if you don't need them any
 longer. Just select one or more objects and press \button{Remove}.
-If you have a very weak computer, you may want to reduce the number of
```

Comment:
"very weak" is better wording here, because "slow" maybe PC with 16-cores modern CPU for subjective reasons...